plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala

```diff
@@ -78,57 +78,58 @@ object ScalafmtPlugin extends AutoPlugin {
 
   val globalInstance = Scalafmt.create(this.getClass.getClassLoader)
 
+  private def withMaybePlural(count: Int, singular: String) =
```

Comment:
There is simpler naming for this function: `private def plural(count: Int, word: String): String = ...`